helm/publisher/templates/article-page-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}--article-page
+  labels:
+    {{- include "publisher.labels" . | nindent 4 }}
+    app.kubernetes.io/component: article-page
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: {{ .Release.Name }}--article-page.libero.pub
```

Comment:
The reviewer-staging deploy has this the other way round: `https://libero-reviewer--staging.elifesciences.org`. Do we have a convention/preference here?

Reply:
This is actually the same convention as this resolves to:
```
http://publisher--test--article-page.libero.pub/
```
`publisher--test` is the equivalent of `libero-reviewer--staging`, but since this chart has more than one public hostname, we add another `article-page` suffix stating the `app.kubernetes.io/component`.